tests/helpers/common/Chocolatey/Get-ChocolateyVersion.ps1

```diff
@@ -8,7 +8,7 @@
     [CmdletBinding()]
     param()
     if (-not $script:runningVersion) {
-        [NuGet.Versioning.NuGetVersion]$script:runningVersion = ((Invoke-Choco --version).Lines | Where-Object { $_ -NotMatch "please upgrade" }) -join '`r`n'
+        [NuGet.Versioning.NuGetVersion]$script:runningVersion = ((Invoke-Choco --version).Lines | Where-Object { $_ -NotMatch "please upgrade" } | Select-Object -Last 1) -join '`r`n'

```

Comment:
It's needed because without it, we think the version contains the compatibility warning, and so our version checks don't actually work. See the below screenshot for what is output with the different scenarios.

<img width="1019" alt="2023-06-26_10-11-23" src="https://github.com/chocolatey/choco/assets/30301021/17876d7e-76ef-4e35-807f-847a490db5d6">